src/bundle/Resources/views/admin/search/search.html.twig

```diff
@@ -0,0 +1,32 @@
+{% extends "EzPlatformAdminUiBundle::layout.html.twig" %}
+
+{% trans_default_domain 'search' %}
+
+{% block pageTitle %}{% endblock %}
+
+{% block content %}
+    <div id="react-udw"></div>
+    <div class="row align-items-stretch ez-main-row">
+
+        {% block sidebar %}
+            {{ parent() }}
+        {% endblock sidebar %}
+
+        <div class="col-sm-10 px-0">
+            <section class="container mt-5">
+                {% include '@EzPlatformAdminUi/parts/page_title.html.twig' with { title: 'search.headline'|trans|desc('Search'), iconName: 'search' } %}
+
+                {% include '@EzPlatformAdminUi/admin/search/search_form.html.twig' with { form: form } %}
+            </section>
+        </div>
+    </div>
+{% endblock %}
+
+{% block title %}{{ 'search.perform'|trans }}{% endblock %}
+
+{% block javascripts %}
+    {% javascripts
+    'bundles/ezplatformadminui/js/scripts/udw/browse.js' %}
```

Comment:
This JS is always needed with the sidebar, if we have sidebar in `layout.html.twig` this JS should be loaded in `layout.html.twig`